lib/api/core/plugins/pluginsManager.js

```diff
@@ -1046,6 +1047,29 @@ class PluginsManager {
             : {}
         };
 
+      // load customs errors configuration file
+      if (!_.has(plugin.manifest.raw, 'errors')) {
```

Comment:
Nit: `lodash.has` is usefull when checking a path, here you can just check `plugin.manifest.raw.errors`